e2e/.percy.ci.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+snapshot:
+  widths: [1024]
```

Comment:
this is an awesome change. Let me calculate the monthly usage of the snapshots and we may just want to run this in a single width for now